es-ds-components/package.json

```diff
@@ -8,7 +8,10 @@
     "scripts": {
         "test": "echo \"Error: no test specified\" && exit 1",
         "lint": "eslint .",
```

Comment:
How would you feel about using the same structure on the linting side of things and the formatting side of things? i.e. switch the existing "lint" script to be called "lint:eslint" and create a new "lint" script which invokes "lint:eslint" and "lint:prettier"? That way, the `make lint` command (which is invoked by the PR CI) will catch prettier errors in addition to eslint. On a related note, could we add the branch [here](https://github.com/EnergySage/es-ds/blob/main/.github/workflows/ci.yml#L11) so that the linting runs when raising a PR against the Vue branch?

Reply:
All that makes sense, I'd be inclined to defer the CI change to another PR as this was sort of a 0-point quick thing